Tests/Remora.Discord.Caching.Tests/Remora.Discord.Caching.Tests.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Remora.Tests.Sdk">
+    <ItemGroup>
+        <ProjectReference Include="..\..\Backend\Remora.Discord.API\Remora.Discord.API.csproj" />
```

Comment:
```suggestion
```
Obsolete reference as it will be pulled by `Remora.Discord.Caching` transitively.